rootfs-builder/rootfs.sh

```diff
@@ -531,6 +583,26 @@ parse_arguments()
 
 	shift $(($OPTIND - 1))
 	distro="$1"
+	arch=$(uname -m)
+
+	if [ "${distro}" == "alpine" -o "${distro}" == "euleros" ]; then
+		if [ "${RUST_AGENT}" == "yes" ]; then
+			die "rust agent cannot be built on ${distro}.
+alpine: only has stable/nightly-x86_64-unknown-linux-musl toolchain. It does not support proc-macro compilation.
+euleros: 1. Missing libstdc++.a
+         2. kernel is 3.10.x, there is no vsock support
+You can build rust agent on your host and then copy it into
+image's rootfs(eg. rootfs-builer/rootfs/usr/bin), and then
+use image_builder.sh to build image with the rootfs. Please
+refer to documentation for how to use customer agent."
+		fi
+	fi
+
+	if [ "${RUST_AGENT}" == "yes" -a "${arch}" == "ppc64le" ]; then
```

Comment:
interesting ...

cc @nitkon

Reply:
@devimc @yyyerbo:  Rust is not supported on Power.

Reply:
@nitkon Thanks for verifying on the toolchain

Reply:
OOI, what do you mean by "supported" @nitkon - it seems to be a rust Tier 2 platform?

- https://forge.rust-lang.org/release/platform-support.html

It would of course be perfect if ppc64le, s390x and aarch64 could become Tier 1.

/cc @alicefr, @jschintag, @Pennyzct, @Weichen81.

Reply:
hi, sorry I missed this issue. I'm also not sure if this is going to work without problem on s390x. There're some issue on Z for rust. I'll have a deeper look :)

Reply:
Could you please skip that for `s390x` arch too?

Reply:
@yyyeerbo - I think you still need to change the test to also exclude s390x here.